Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -99,7 +97,22 @@
     'profile-setting': 'ProfileSetting',
     'audit-comment': 'AuditComment'
 }
-
+ELEM_TO_CONTEXT = {
+    'source': 'Source',
+    'destination': 'Destination',
+    'application': 'Application',
+    'action': 'Action',
+    'category': 'Category',
+    'description': 'Description',
+    'disabled': 'Disabled',
```

Comment:
Why do you want to add new dict instead of adding the source-user value to the SECURITY_RULE_ARGS dict?

Reply:
Because the `SECURITY_RULE_ARGS` dict was made for a different command and which parses over the the keys. Adding keys will break the behavior.